src/main/java/com/letsparty/mapper/PostMapper.java

```diff
@@ -1,12 +1,28 @@
 package com.letsparty.mapper;
 
 import org.apache.ibatis.annotations.Mapper;
+import org.apache.ibatis.annotations.Param;
 
+import com.letsparty.dto.BeginEndPostNo;
+import com.letsparty.dto.SimplePostDto;
 import com.letsparty.vo.Post;
 
 @Mapper
 public interface PostMapper {
 
 	void insertPost(Post post);
 
+	int getLastPostNoByPartyNo(int partyNo);
```

Comment:
```suggestion
	Integer getLastPostNoByPartyNo(int partyNo);
```
바꾸고 컨트롤러에서 null처리하면 좋을 것 같습니다.